istio/tests/test_e2e.py

```diff
@@ -27,5 +27,3 @@ def test_e2e(dd_agent_check):
             aggregator.assert_metric(metric, at_least=0)
         else:
             aggregator.assert_metric(metric)
-
-    aggregator.assert_all_metrics_covered()
```

Comment:
Why was this assert removed? Just want to understand. Are some of the collected metrics too intermittent?

Reply:
It was intermittent with gke, and I don't see it at all in kind.